dist/index.html

```diff
@@ -4,31 +4,92 @@
+            </p>
+
+            <p>
+              No matter where she has been, she's produced superior work because she dedicates making sure each and every piece stands to return becuase of the attention to detail and the stress-free process that she prides herself on.
+            </p>
+          </div>
+        </div>
+
+        <div class="seeking">
+          <h2>SEEKING-A-REVIEW?</h2>
+          <hr>
+          <div class="seeking-text">
+            <ul>
+              <li>
+                “We had the most amazing wedding invitations created by Seek-A-Design!
+                It was cheaper than the prices I found online, and better quality. I received
+                so many compliments that I’m using them again for an upcoming party I’m
+                throwing. We can’t say enough!” - Angela L.
+              </li>
+
+              <li>
+                “Seek-A-Design is a one stop shop for any of your graphic design needs.
+                The owner brings your vision to life and delivers a finished product
+                better than you ever could have imagined. Seek-A-Design beats the
+                competition in quality, prices and overall experience. There's a reason
+                I've returned...5 times!” - Katie G.
+              </li>
+
+              <li>
+                “Seek-a-Design helped create invitations, place-cards, signage and thank you
+                cards for my great-grandmothers centennial celebration. Lyandra was patient,
+                collaborative and took into consideration all of our wants while still being
+                creative. We received so many complements on the quality and graphics
+                from attendees. I will definitely use Seek-a-Design for my future event
+                and signage needs!” - Valentina C.
+              </li>
+
+              <li>
+                “I had Seek-A-Design create some brochures for my company with a
+                fairly close timeframe, not only did she knock it out of the park that
+                at our latest showcase we had other companies coming by to comment
+                on how incredible they were compared to their own. I would recommend
+                them anytime for anything from work related printing, to business cards
+                and invitations. A++.” - Tom F.
+              </li>
+            </ul>
+          </div>
+        </div>
       </div>
-      <div class="seeking">
-        <h2>SEEKING-A-REVIEW?</h2>
-        </br>
-        <p>Seek-A-Design was a long-time dream of owner and designer, Lyandra Sica (seek•uh). Using her last name, she developed the play-on-words into her business which has sprouted vivdly within the last few years. Within those years, her hard work and determination has allowed her dream to turn into the reality of being a go-to designer and printing expert.</p>
+
+      <div class="smicons">
+
```

Comment:
Nice, love that you left room for this section. Let's fill this in with some HTML for each icon. 

Fortunately, we have a team member who's already built a system of 3 icons, each on a solid circular background. Take a look at the code in the `landon` branch to help you get started with this section. 

*Note:* There is no shame is collaborating with your team mates. In fact, it's better to re-use already existing code than it is to re-write it! The CSS styling may be a bit different here (because the icons are smaller), but the HTML structure will likely be quite similar. 👍

Reply:
Okay, so I came close but I'm still having difficulties arranging/aligning with grid. I'll spend some more time reviewing some tutorials tomorrow.